src/com/sun/ts/lib/harness/keyword.properties

```diff
@@ -181,7 +195,7 @@ com/sun/ts/tests/jstl = jstl javaee javaee_web_profile
 com/sun/ts/tests/jta = jta javaee javaee_web_profile
 com/sun/ts/tests/jta/ee/txpropagationtest = jta javaee
 com/sun/ts/tests/jta/ee/usertransaction = jta javaee javaee_web_profile
-com/sun/ts/tests/rmiiiop = rmiiiop javaee
+com/sun/ts/tests/rmiiiop = rmi_iiop corba 
```

Comment:
:+1: 
So these tests are not enabled by default for javaee, instead they are only enabled if rmi-iiop or corba are specified.